src/blk/pmem/PMEMDevice.cc

```diff
@@ -28,6 +28,12 @@
 #include "common/debug.h"
 #include "common/blkdev.h"
 
+#if defined(HAVE_LIBDML)
+#include <dml/dml.hpp>
+using execution_path = dml::automatic;
+using namespace std;
```

Comment:
why is this `using namespace std` necessary?

Reply:
Let me remove it.